tensorboard/webapp/widgets/card_fob/card_fob_types.ts

```diff
@@ -25,31 +25,31 @@ export interface TimeSelection {
  * Only used for internal analytics.
  */
 export enum TimeSelectionAffordance {
-  NONE,
+  NONE = 'NO_AFFORDANCE',
```

Comment:
Note that the string values of `TimeSelectionAffordance.NONE` and `TimeSelectionToggleAffordance.NONE` are intentionally not set to `"NONE"` so that they are different in case we wanted to handle them differently in the future.